src/config/schema.ts

```diff
@@ -573,42 +523,6 @@ export const configSchema = {
         swapEnabled: true,
       },
     },
-    kratosConfig: {
-      type: "object",
-      properties: {
-        publicApi: { type: "string" },
-        adminApi: { type: "string" },
-        corsAllowedOrigins: {
-          type: "array",
-          items: { type: "string" },
-          uniqueItems: true,
-        },
-      },
-      required: ["publicApi", "adminApi", "corsAllowedOrigins"],
-      additionalProperties: false,
-      default: {
-        publicApi: "http://localhost:4433",
-        adminApi: "http://localhost:4434",
-        corsAllowedOrigins: ["http://localhost:3000"],
-      },
-    },
-    oathkeeperConfig: {
-      type: "object",
-      properties: {
-        urlJkws: { type: "string" },
-        decisionsApi: { type: "string" },
-      },
-      required: ["urlJkws", "decisionsApi"],
-      additionalProperties: false,
-      default: {
-        urlJkws: `http://${
-          process.env.OATHKEEPER_HOST ?? "oathkeeper"
-        }:4456/.well-known/jwks.json`,
-        decisionsApi: `http://${
-          process.env.OATHKEEPER_HOST ?? "oathkeeper"
```

Comment:
Btw, I was caught out by exactly this in #3020 and #3022 where I couldn't figure out where oathkeeper settings were coming from and hadn't thought to check that an environment variable was being pulled into yaml config

Reply:
yup this PR should clean this all up